### PR TITLE
Greenhouse biomass

### PR DESCRIPTION
**Pull request by jessicaguo**
I reorganized the biomass subfolder of parameters_processing. 02_organize_greenhouse_outdoor_biomass.Rmd now organizes the raw biomass Excel file and outputs greenhouse_outdoor_biomass.Rdata in the sentinel-detection/data/cleaned_data/biomass folder. 
The ANOVA and plotting script is now renamed Chamber_effect_biomass.Rmd

02_parameter_upload.Rmd has also been revamped to filter the organized .Rdata outputs and calculate SLA more efficiently.

### CODE REVIEW

File: parameters_processing/02_parameter_upload.Rmd [hunk 1]
```diff
@@ -169,153 +169,117 @@ These are the final columns:
...
+         SLA = leaf_area_m2 / leaf_DW_kg,
+         Treatment = "outdoor JollyG soil",
+         local_datetime = harvest_date) %>% 
+  select(local_datetime, Treatment, SLA) 
 ```
 
-Combineand format all SLA values for upload. 
+Combine and format all SLA values for upload. 
 ```{r}
 sla <- bind_rows(sla_hightemp, sla_control, sla_gh, sla_5cm, sla_JollyG) %>%
```
Comment (KristinaRiemer) [inline parameters_processing/02_parameter_upload.Rmd:266]: Comparing the old version to this, they were identical except for a one day shift for the outdoor data. That seems like a minor difference, just wanted to document it. 
Screen Shot 2021-01-29 at 9 14 40 AM

**Review by KristinaRiemer — APPROVED** [review]
This looks great! Way nicer and easier to follow with all the biomass data collated.

File: parameters_processing/biomass/02_organize_greenhouse_outdoor_biomass.Rmd [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+```{r data}
+data_path <- "~/sentinel-detection/data/raw_data/biomass/field_and_greenhouse_experiments.xlsx"
+sheets_names <- excel_sheets(data_path)
+```
+
+## Experiments
+
+The first greenhouse experiment was conducted on ME034V-1 in GH2B under the temperature conditions of 22/20 day/night temperature. 4 planting densities were tested (5 cm, 15 cm, 25 cm, and pot). 
+```{r 1st}
+gh_1 <- read_excel(data_path, sheets_names[16], n_max = 72) %>%
```
[the remark this answers is not in the document]
Comment (jessicaguo) [reply parameters_processing/biomass/02_organize_greenhouse_outdoor_biomass.Rmd:37]: Yes, and now there've been 2 new tabs added. Updated.

File: parameters_processing/02_parameter_upload.Rmd (hunk 1 above)
Comment (jessicaguo) [reply parameters_processing/02_parameter_upload.Rmd:266]: Noted. Good eye!

**Pull request merged**